### PR TITLE
workspace focus fix

### PR DESCRIPTION
**Pull request by thesahindia**
### Details

Fixed highlighting issue in workspace
### Fixed Issues

$ https://github.com/Expensify/App/issues/7036

### QA Steps

1. Open app
2. Navigate to workspace
3. Click any page and navigate back out
### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

#### Web

https://user-images.githubusercontent.com/83179295/148269340-e6f25afe-0e0a-44c5-9627-87f39ed7fbce.mov

### CODE REVIEW

**Comment by rushatgabhane** [comment]
Tests well on mobile and web. LGTM!

@Beamanator all yours!

**Review by rushatgabhane — DISMISSED** [review]
Tests well on mobile and web. LGTM!

@Beamanator all yours!

File: src/pages/workspace/WorkspaceInitialPage.js [hunk 1]
```diff
@@ -24,9 +22,6 @@ import withFullPolicy, {fullPolicyPropTypes, fullPolicyDefaultProps} from './wit
 import * as PolicyActions from '../../libs/actions/Policy';
 
 const propTypes = {
-    /** Whether the current screen is focused. */
-    isFocused: PropTypes.bool.isRequired,
-
     ...fullPolicyPropTypes,
     ...withLocalizePropTypes,
     ...windowDimensionsPropTypes,
```
Comment (Beamanator) [inline src/pages/workspace/WorkspaceInitialPage.js:27]: Correct me if I'm wrong, but I believe this & related imports can be removed too

**Review by Beamanator — CHANGES_REQUESTED** [review]

File: src/pages/workspace/WorkspaceInitialPage.js (hunk 1 above)
Comment (rushatgabhane) [reply src/pages/workspace/WorkspaceInitialPage.js:27]: We can get rid of windowDimensions for sure.
But we need the policy and localize prop types. (Used for props.translate, props.policy)

Comment (thesahindia) [reply src/pages/workspace/WorkspaceInitialPage.js]: Done!

**Comment by rushatgabhane** [comment]
@Beamanator good for review.

File: src/pages/workspace/WorkspaceInitialPage.js (hunk 1 above)
Comment (Beamanator) [reply src/pages/workspace/WorkspaceInitialPage.js]: awesome thanks!

**Review by Beamanator — APPROVED** [review]
Looks great 💪

**Pull request merged**